sky/jobs/server/core.py

```diff
@@ -601,6 +617,17 @@ def queue(refresh: bool,
             does not exist.
         RuntimeError: if failed to get the managed jobs with ssh.
     """
+    if limit is not None:
+        if limit < 1:
+            raise ValueError(f'Limit must be at least 1, got {limit}')
+        if offset is None:
+            offset = 1
```

Comment:
Why offset start from 1 instead of 0?

Reply:
This aligns with the page number in the frontend.